ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/winc/provision/cucushift-installer-rehearse-aws-ipi-ovn-winc-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-winc-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ovn-conf
+  - ref: ovn-conf-hybrid-manifest
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - ref: ssh-bastion
+  - ref: operatorhub-subscribe
+  - ref: windows-conf-operator
+  - ref: ipi-conf-aws-windows-machineset
+  - ref: openshift-qe-winc
```

Comment:
Sounds like the `openshift-qe-winc` is for test purpose? Is it a MUST for the whole install workflow?

Reply:
It is indeed for the test purpose, to prepare some stuff that the test cases expect. However, if not added here, where do you suggest to add them? Without this ref being run all test cases will fail...

Reply:
At the same time, it does not harm the cluster at all..it's just creating a configmap with data needed during testing as well as some Windows and Linux workloads (everything under the winc-test namespace)

Reply:
got it.